.github/workflows/documentation.yml

```diff
@@ -0,0 +1,50 @@
+name: documentation
+
+on:
+  pull_request:
+    branches: [documentation]
+  push:
+    branches: [documentation]
+
+jobs:
+ checks:
+   if: github.event_name != 'push'
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v1
+     - uses: actions/setup-node@v1
+       with:
+         node-version: '12.x'
+         run: |
+          npm ci
+          npm run build
+ gh-release:
```

Comment:
I would just call that release (or gh-pages-release, if you prefer).

Reply:
I'll make a BLI to rename this. Works?

Reply:
I want to make it on the local branch, not the fork.

Reply:
I really don't think this is relevant enough to make a task in the backlog for it ;) I don't know what you mean by your last comment, but this was merely a suggestion, so no worries.

Reply:
After I merge this I can update it via a local branch as currently I have to do it via my fork.